internal/postprocessor/main.go

```diff
@@ -268,3 +246,177 @@ func docURL(cloudDir, importPath string) (string, error) {
 	pkgPath := strings.TrimPrefix(strings.TrimPrefix(importPath, mod), "/")
 	return "https://cloud.google.com/go/docs/reference/" + mod + "/latest/" + pkgPath, nil
 }
+
+func (c *config) AmendPRDescription(ctx context.Context) error {
+	log.Println("Amending PR title and body")
+	pr, err := c.getPR(ctx)
+	if err != nil {
+		return err
+	}
+	prTitle := pr.Title
+	prBody := pr.Body
+	newPRTitle, newPRBody, err := c.processCommit(*prTitle, *prBody)
+	if err != nil {
+		return err
+	}
+	if err := c.writePRCommitToFile(newPRTitle, newPRBody); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *config) processCommit(title, body string) (string, string, error) {
+	var newPRTitle string
+	var commitTitle string
+	var commitTitleIndex int
+
+	bodySlice := strings.Split(body, "\n")
+	for index, line := range bodySlice {
+		if strings.Contains(line, "[REPLACEME]") {
```

Comment:
nit: might be nice to pull out REPLACE me into a const

Reply:
Done. Also updated regex patterns.